### ISSUE TITLE
Idea: Oredict support for taming/breeding animals

### ISSUE THREAD

**Issue by Sunconure11** [itsmeow/betteranimalsplus#110]
Mainly for a lot of the meats and some of the plants, I was inspired by Animania implementing a similar feature for feeding animals, so why not allow oredicts as valid definitions of items for this context?

**Comment by itsmeow** [comment]
We have config support for taming items.

**Comment by Sunconure11** [comment]
I'm aware, I'm referring to oredict entries being valid for configuration files.

**Comment by williambl** [comment]
While working on this I realised that taming item configuration is totally broken - every time it tries to load the config, Forge says that the tame items are invalid and resets them:

```
[20:31:59] [Server thread/WARN] [ne.mi.co.ForgeConfigSpec/CORE]: Incorrect key lammergeier.tameItems was corrected from [minecraft:bone] to [minecraft:bone]
[20:31:59] [Server thread/WARN] [ne.mi.co.ForgeConfigSpec/CORE]: Incorrect key feralwolf.tameItems was corrected from [minecraft:bone] to [minecraft:bone]
[20:31:59] [Server thread/WARN] [ne.mi.co.ForgeConfigSpec/CORE]: Incorrect key coyote.tameItems was corrected from [minecraft:rabbit, minecraft:chicken, betteranimalsplus:pheasantraw, minecraft:cooked_rabbit, minecraft:cooked_chicken, betteranimalsplus:pheasantcooked] to [minecraft:rabbit, minecraft:chicken, betteranimalsplus:pheasantraw, minecraft:cooked_rabbit, minecraft:cooked_chicken, betteranimalsplus:pheasantcooked]
```

**Comment by itsmeow** [comment]
yes it corrects them to themselves. It's just forge being dumb. It still works afaik

**Comment by williambl** [comment]
yeah but if you change it at all it gets reset to the original.

**Comment by itsmeow** [comment]
Solved on 1.14, denote tags with '#yourtag', 1.12 not added.
